src/autoIncrement.ts

```diff
@@ -94,18 +100,28 @@ export function AutoIncrementID(schema: mongoose.Schema<any>, options: AutoIncre
 
     if (!model) {
       logger.info('Creating idtracker model named "%s"', opt.trackerModelName);
-      model = this.db.model(opt.trackerModelName, IDSchema, opt.trackerCollection);
+      model = this.db.model(opt.trackerModelName, IDSchema);
+      model.findOne({ model: opt.model, field: opt.field } as AutoIncrementIDTrackerSpec).lean()
+        .then((counter: AutoIncrementIDTrackerSpec) => {
+          if (!counter) {
+            model.create({
+              model: opt.model,
+              field: opt.field,
+              count: opt.startAt - opt.incrementBy
```

Comment:
why decrement it?

Reply:
if we want it to start from 10 means, we have to decrement by one and it will get stored in a separate collection with the model we have given. Then when we save any docs it will start count from 10.

Reply:
right forgot about the $inc